### ISSUE TITLE
Support advanced symsrv paths

### ISSUE THREAD

**Issue by DrChat** [microsoft/pdblister#21]
Documentation: https://learn.microsoft.com/en-us/windows-hardware/drivers/debugger/advanced-symsrv-use

**Comment by mstange** [comment]
The symsrv crate has [parsing code for this syntax](https://docs.rs/symsrv/latest/symsrv/fn.parse_nt_symbol_path.html), if you're curious.

**Comment by DrChat** [comment]
Yeah - I had actually opened this ticket after seeing your repository had it implemented correctly.
Tbh I would prefer to use your repository instead of maintaining our divergent copy, but one requirement we have is that PDBs _are_ downloaded to a local cache directory so that the library returns file paths instead of in-memory blobs.
